src/pycrunch/elements.py

```diff
@@ -150,6 +150,39 @@ def __getattr__(self, key):
         raise AttributeError(
             "%s has no attribute %s" % (self.__class__.__name__, key))
 
+    def void(self, key, **members):
+        """Return a dummy Document without populating it via HTTP GET.
+
+        For example, call `root.void("foo").post(entity)` to POST an entity to
+        the "foo" resource without having to first GET the "foo" resource. This
+        can be a useful optimization for large resources that are expensive to GET.
+
+        This returns a Document instance with a `self` attribute, but no
+        additional members other than those provided. It can't, because
+        populating those would require a GET, which this method purposefully
+        avoids. If you later need to populate the Document, simply call its
+        refresh() method to perform the GET.
+
+        The returned Document will have `post`, `put`, `patch`, and `delete`
+        methods, which only need the `self` URL to function properly. If a
+        different Document subclass can be determined from the navigation
+        collection in which the key is found, it will be used and some of
+        its methods may also work properly if they only need the `self` URL.
+        If they need additional members, you must either provide them as keyword
+        arguments to this method, or call .refresh().
+
+        Use this method sparingly, and only with very stable APIs, because it
+        makes assumptions about the media type which would have been returned
+        if had performed an HTTP GET.
```

Comment:
"Use this method sparingly and only with very stable APIs" seems like a strange thing to put in a public-facing API doc.
How is a customer supposed to know what APIs are "very stable"?

Reply:
I'll try to re-word that.